cookbooks/aws-parallelcluster-slurm/files/default/head_node_checks/check_cluster_ready.py

```diff
@@ -112,13 +112,20 @@ def check_deployed_config_version(cluster_name: str, table_name: str, expected_c
 
         missing, incomplete, wrong = _check_cluster_config_items(instance_ids, items, expected_config_version)
 
-        if missing or incomplete or wrong:
+        if incomplete or wrong:
             raise CheckFailedError(
                 f"Check failed due to the following erroneous records:\n"
```

Comment:
When the exception is raised, the warning below is not printed.
Can we add to this line "missing records are not counted for the failure"?